sci-physics/fastjet/fastjet-3.4.0.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit autotools flag-o-matic fortran-2 python-r1
+
+DESCRIPTION="A software package for jet finding in pp and e+e- collisions"
+HOMEPAGE="http://fastjet.fr/"
+SRC_URI="http://fastjet.fr/repo/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cgal doc examples python +plugins +contrib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+sys-devel/gcc:=[fortran]
+contrib? ( sci-physics/fastjet-contrib )
+cgal? ( sci-mathematics/cgal:=[shared(+)] )
+plugins? ( sci-physics/siscone:= )
+python? ( ${PYTHON_DEPS} )
```

```suggestion
	sys-devel/gcc:=[fortran]
	contrib? ( sci-physics/fastjet-contrib )
	cgal? ( sci-mathematics/cgal:=[shared(+)] )
	plugins? ( sci-physics/siscone:= )
	python? ( ${PYTHON_DEPS} )
```
I'm not 100% sure, but I think we can depend on `virtual/fortran` instead of on `sys-devel/gcc[fortran]`.
In any case it is probably a BDEPEND and not a DEPEND. BDEPEND is for things that we execute when emerging the package, DEPEND is for things that have to be present but are not executed.